.github/workflows/solrj-test.yml

```diff
@@ -0,0 +1,27 @@
+name: SolrJ Tests
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - '.github/workflows/solrj-test.yml'
+      - 'solr/solrj/**'
+
+jobs:
+  test:
+    name: Run SolrJ Tests
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Setup
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Test the SolrJ Package
+      run: ./gradlew solr:solrj:test
```

Comment:
Do we need to consider that this test run won't have our generated gradle properties?

Reply:
I copied the same format as the precommit action, is there something we need to add to use the right properties?